project/MimaExcludes.scala

```diff
@@ -278,6 +278,9 @@ object MimaExcludes {
     // [SPARK-26616][MLlib] Expose document frequency in IDFModel
     ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.mllib.feature.IDFModel.this"),
     ProblemFilters.exclude[IncompatibleResultTypeProblem]("org.apache.spark.mllib.feature.IDF#DocumentFrequencyAggregator.idf")
+
+    // [SPARK-26254][CORE] Extract Hive + Kafka dependencies from Core.
+    ProblemFilters.exclude[MissingClassProblem]("org.apache.spark.deploy.security.HiveDelegationTokenProvider")
```

Comment:
This doesn't compile, does it?

(Add new exclusions at the top. It avoids having to change existing code.)

Reply:
Yeah, doesn't compile. I've seen the trend to add things at the end but better to add as you've suggested to avoid such mistakes.